vars/ohpc-1.3

```diff
@@ -3,18 +3,29 @@
 
 ohpc_slurm_packages:
   control:
-    - "@ohpc-slurm-server"
-    - "slurm-slurmctld-ohpc"
-    - "slurm-example-configs-ohpc"
+    - name: "@ohpc-slurm-server"
+      install: true
+    - name: "slurm-slurmctld-ohpc"
+      install: true
+    - name: "slurm-example-configs-ohpc"
+      install: true
   batch:
-    - "@ohpc-base-compute"
-    - "@ohpc-slurm-client"
+    - name: "@ohpc-base-compute"
+      install: true
+    - name: "@ohpc-slurm-client"
+      install: true
   runtime:
```

Comment:
As per general comment, suggest just keeping these as a list, and adding:

`- "{{ 'lmod-ohpc' if openhpc_module_system_install else '' }}"`